src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,32 @@
+import { useState } from 'react';
 import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [loading, setLoading] = useState(true);
+
   return (
     <>
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
           <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
+            {!loading && <PeopleFilters />}
```

Comment:
Ensure that the `loading` state is set to `false` once the data is fetched. Otherwise, `PeopleFilters` will not be displayed, which might confuse users if the data is already loaded.